experimental/dds/tree2/src/feature-libraries/sequence-field/compose.ts

```diff
@@ -607,7 +644,7 @@ interface ComposeMarks<T> {
 
 /**
  * Returns whether `baseMark` and `newMark` are inverses.
- * It is assumed that both marks are active and `baseMark` is an attach.
+ * It is assumed that both marks are active, `baseMark` is an attach, and `newMark` is a detach.
  */
 function areInverseMoves(
```

Comment:
Consider renaming this function so that one couldn't mistakenly think it will handle the case where baseMark is a detach and newMark is an attach.
Something like `areInverseMovesAtTemporaryLocation` maybe?
Or better yet, capture it in the type system if that doesn't make it too cumbersome for the call sites.